src/instrumentations/ioredis/IORedisInstrumentation.ts

```diff
@@ -0,0 +1,39 @@
+import { CommonUtils, ScrubContext } from '@lumigo/node-core';
+import { Span } from '@opentelemetry/api';
+import { IORedisInstrumentation } from '@opentelemetry/instrumentation-ioredis';
+import { getSpanAttributeMaxLength } from '../../utils';
+import { Instrumentor } from '../instrumentor';
+
+export default class LumigoIORedisInstrumentation extends Instrumentor<IORedisInstrumentation> {
+  getInstrumentedModule(): string {
+    return 'ioredis';
+  }
+
+  getInstrumentation(): IORedisInstrumentation {
+    return new IORedisInstrumentation({
+      dbStatementSerializer: function (cmdName, cmdArgs) {
+        const statement = [cmdName, ...cmdArgs].join(' ');
+        return CommonUtils.payloadStringify(
+          statement,
+          ScrubContext.HTTP_REQUEST_BODY,
+          getSpanAttributeMaxLength()
+        );
```

Comment:
though this is a cool usage of `dbStatementSerializer`, you can simplify your implementation by moving this code to the response hook